common/js/server.js

```diff
@@ -0,0 +1,462 @@
+/* global owaspPasswordStrengthTest, TextEncoder */
```

Comment:
Well since it's a file I created and written based on JS code I extracted from `default-views/account/register-form.hbs` I don't know if we need to put the license since we don't have other file with a license header. If so, should I copy/paste the content of `LICENSE`?

Reply:
Ah this file is your work entirely? Wasn't sure if it was a generic password tester.